README.md

```diff
@@ -15,18 +15,14 @@ jobs:
     runs-on: ubuntu-latest
     name: Upload robot
     steps:
-      - uses: actions/checkout@master
-      - name: Use Node.js 12.x
-        uses: actions/setup-node@v1
-        with:
-          node-version: 12.x
+      - name: Checkout files from repository
```

Comment:
If this is step is mandatory as we thought, maybe add comment here?

Reply:
It's not really mandatory to use the action. The user might download the files from somewhere else before triggering the update action. It's good to have this step in the example because checking out code and then uploading that is the most common use case.